pkg/apitype/policy.go

```diff
@@ -54,6 +54,10 @@ type RequiredPolicy struct {
 
 	// Where the Policy Pack can be downloaded from.
 	PackLocation string `json:"packLocation,omitempty"`
+
+	// Maps Policy names to the configuration to use for that policy.
+	// If Policy name is not in the map, use the default configuration.
+	Configuration map[string]PolicyConfiguration `json:"configuration"`
```

Comment:
My understanding is that the Golang documentation convention is to prefix the comment with the name of the field/function/type being documented. For example, starting with the word "Configuration" here. I assume this is getting past the linter, so is this optional? Have I been doing it all wrong? Just wondering what the standard we want for struct field comments to be.

https://blog.golang.org/godoc-documenting-go-code